Editor/XML_ActionItems.cpp

```diff
@@ -3,7 +3,6 @@
 	#include "Editor All.h"

```

Comment:
This include should be removed as well, no?

Reply:
5d303249966458896c10d5034e8b31079f9daf50 is it